src/benchmark/file_io_write_micro_benchmark.cpp

```diff
@@ -214,8 +220,95 @@ void FileIOWriteMicroBenchmarkFixture::mmap_write_benchmark(benchmark::State& st
   }
 }
 
+/*
+ * Performs a benchmark run with the given parameters. 
+ * 
+ * @arguments:
+ *      state: the benchmark::State object handed to the called benchmarking function.
+ *      flag: The mmap flag (e.g., MAP_PRIVATE or MAP_SHARED).
+ *      data_access_mode: The way the data is written.
+ *                  (-1)  No data access
+ *                  (0)   Sequential
+ *                  (1)   Random
+ *      file_size: Size argument of benchmark.
+*/
+void FileIOWriteMicroBenchmarkFixture::umap_write_benchmark(benchmark::State& state,
+                                                            int data_access_mode, const int32_t file_size) {
```

Comment:
in #36  we introduce an enum for the data_access_mode flags - please also do this here (or we can do it together after the merge)